reckless_lib/src/plugin_manager.rs

```diff
@@ -12,7 +12,7 @@ pub trait PluginManager {
 
     /// install a sequence of plugin or return an error if somethings happens.
     // FIXME: what happens if only one plugin fails?
-    async fn install(&mut self, plugins: &[&str]) -> Result<(), RecklessError>;
+    async fn install(&mut self, plugins: Vec<String>) -> Result<(), RecklessError>;
```

Comment:
```suggestion
    async fn install(&mut self, plugins: &Vec<String>) -> Result<(), RecklessError>;
```

there is no reason to pass a copy of the value